README.md

```diff
@@ -105,6 +105,9 @@ Other methods an `Expert` needs to provide:
 
 ## Release notes
 
+### 0.15.0-dev
```

Comment:
The addition of a feature can be done in a 0.0.x step, as long as the component is in the 0.x range.